.gn

```diff
@@ -167,11 +167,11 @@ exec_script_whitelist =
       "//tools/gritsettings/BUILD.gn",
     ]
 
+# TODO(b/378960295): Convert python scripts to GN actions.
```

Comment:
This is not what this allowlist is allowing, it's for exec_script calls which would likely not be converted to actions but be handled some other way.

Reply:
modified comment to reflect what we want to do